src/api.test.ts

```diff
@@ -39,7 +39,8 @@ function byMajorMinorPatch(a: VersionNumber, b: VersionNumber): number {
 
 async function getChangelogVersions(): Promise<VersionNumber[]> {
     const files = await readdir("doc/changes")
-    return files.map(name => name.match(/^changes_(\d+)\.(\d+)\.(\d+)\.md$/))
+    const changesFilePattern = RegExp(/^changes_(\d+)\.(\d+)\.(\d+)\.md$/);
```

Comment:
It is usually a good idea to add an example what the regex would match.

Reply:
I will add the comment in the next version